src/lib/features/segment/segment-read-model.ts

```diff
@@ -75,4 +80,28 @@ export class SegmentReadModel implements ISegmentReadModel {
             segmentId: row.segment_id,
         }));
     }
+
+    async getActive(): Promise<ISegment[]> {
```

Comment:
copied from the store, will delete from the store after we migrate all usages (next PR)